docs/open-data/README.md

```diff
@@ -27,6 +28,26 @@ Reporting is the life-blood of any FinOps initiative. And your reports are only
 
 <br>
 
+## 📏 Pricing units
+
+Microsoft Cost Management uses the `UnitOfMeasure` column to indicate how each charge is measured. This can be in singular or distinct units or can be grouped into chunks based on applicable block pricing rules. As a string, the `UnitOfMeasure` column can be challenging to parse and handle all the different permutations and inconsistencies. The Pricing units file provides a list of values you may find within common cost-related datasets (e.g., Cost Management exports and price sheets) along with their related distinct unit and scaling factor to compare pricing to usage units.
+
+Sample data:
+
+| UnitOfMeasure      | MeterCount | UsageToPricingRate | DistinctUnits |
+| ------------------ | ---------- | ------------------ | ------------- |
+| `1 Hour`           | 116073     | 1                  | Hours         |
+| `10000 GB`         | 342        | 10000              | GB            |
+| `150 Hours`        | 4          | 150                | Hours         |
+| `200 /Hour`        | 4          | 200                | Units/Hour    |
+| `5 GB`             | 16         | 5                  | GB            |
+| `5000000 Requests` | 1          | 5000000            | Requests      |
+| `744 Connections`  | 26         | 744                | Connections   |
+
+[Download](https://github.com/microsoft/finops-toolkit/releases/latest/download/PricingUnits.csv){: .btn .fs-5 .mb-4 .mb-md-0 .mr-4 }
+
+<br>
+
```

Comment:
It's great to have insight into the full list of values available in the unit of measure column, already broken down into units and scaling factors - ready for FOCUS converter. 

Additionally, it might be useful to include an additional column to indicate if a value is applicable for either 

- `EA` / `MCA` / `both` 
- or `legacy` / `modern` / `both`

whichever fits better.

Reply:
Done. I added an AccountTypes column that indicates which account types it's used for. Values are `EA`, `MCA`, and `MCA, EA`.